scripts/update-browser-releases/chrome.ts

```diff
@@ -171,9 +169,22 @@ export const updateChromiumReleases = async (options) => {
   //
   // Write the update browser's json to file
   //
+
+  // We want the 'release' object to be after 'type' and 'upstream'
+  // The others in lexicographic order.
+  // This is an array of array because there may be different orders
+  // at different ordering
+  // Non-listed entries will be put in the lexcicographical order.
+  const orders = [
```

Comment:
I think this could be hidden away in the sort function. No need to copy it in all browser-specific files. We want it to be the same everywhere.

Reply:
Yep, good point. Fixed.